e2e_report_test.go

```diff
@@ -55,6 +55,78 @@ func TestE2EReport_Names(t *testing.T) {
 	assert.Contains(t, reporter.recorded, "RequestExample")
 }
 
+func TestE2EReport_Formatting(t *testing.T) {
```

Comment:
What do you think about renaming it to TestE2EReport_Values?

We have TestE2EReport_Aliases that checks formatting of aliases and TestE2EReport_Names that checks formatting of names. And this test mostly checks formatting of Actual / Expected / Reference / Delta values (AssertionValue).